pom.xml

```diff
@@ -1,219 +0,0 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">

```

Comment:
This POM seems to be outdated. Could we remove it?

Reply:
I _guess_. We only publish this to our own Artifactory so I believe it should work without this file.

Reply:
Yes.